<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettings</class>
 <widget class="QDialog" name="GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>428</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>650</width>
    <height>300</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>LinPSK General Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QLabel" name="CallsignL">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Callsign</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="Callsign"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="UTCL">
     <property name="text">
      <string>Time Offset to UTC</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="UTC">
     <property name="minimum">
      <number>-12</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>178</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Locator</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="myLocator"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Date Format</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="dateFormat">
     <property name="whatsThis">
      <string>describes the format displaying the date within qsodata </string>
     </property>
     <item>
      <property name="text">
       <string>dd.MM.yyyy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>yyyy-MM-dd</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>178</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>283</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QRadioButton" name="autoDate">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Set Date qso saving</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QRadioButton" name="SlashedZero">
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Display Zero as Slashed  Zero</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QRadioButton" name="autoCrLf">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Send CR LF on enter</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="commParams">
     <property name="title">
      <string>Rig Comm Parameter</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Baudrate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="baudRate">
        <property name="toolTip">
         <string>Select Baudrate</string>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Handshake</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="handShake">
        <property name="toolTip">
         <string>Select Handshake</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="maxVisibleItems">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hardware</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XonXoff</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="Demomode">
        <property name="text">
         <string>Demomode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="3" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Logging</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>File logging</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="fileLog">
           <property name="text">
            <string>Log to File</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Working Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Directory"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Filename</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="QsoFile">
           <property name="text">
            <string>QSO.adif</string>
           </property>
           <property name="cursorPosition">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>LinLog logging</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="LinLog">
           <property name="text">
            <string>Log to LinLog</string>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>LinLog Host Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Host">
           <property name="text">
            <string>localhost</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Port">
           <property name="minimum">
            <number>1024</number>
           </property>
           <property name="maximum">
            <number>64000</number>
           </property>
           <property name="value">
            <number>8080</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Rig Devices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="Rignumber">
          <property name="text">
           <string>Rig number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="modelNr">
          <property name="toolTip">
           <string>Set Rig number as defined in hamlib</string>
          </property>
          <property name="inputMask">
           <string extracomment="Modelnumber from hamlib">00999</string>
          </property>
          <property name="text">
           <string extracomment="Hamlibs rig model number"/>
          </property>
          <property name="maxLength">
           <number>5</number>
          </property>
          <property name="cursorPosition">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>PTT device</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Rigdevice">
          <property name="text">
           <string>Rig device</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="pttDevice"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="rigControl"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QGroupBox" name="SoundDeviceBox">
     <property name="title">
      <string>Sound Device</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Input card or device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Output card or device</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="soundInputDeviceName"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="soundOutputDeviceName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>358</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>Callsign</tabstop>
  <tabstop>UTC</tabstop>
  <tabstop>Demomode</tabstop>
  <tabstop>fileLog</tabstop>
  <tabstop>Directory</tabstop>
  <tabstop>QsoFile</tabstop>
  <tabstop>LinLog</tabstop>
  <tabstop>Host</tabstop>
  <tabstop>Port</tabstop>
 </tabstops>
 <includes>
  <include location="local">parameter.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Demomode</sender>
   <signal>clicked(bool)</signal>
   <receiver>GeneralSettings</receiver>
   <slot>selectDemomode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LinLog</sender>
   <signal>clicked(bool)</signal>
   <receiver>GeneralSettings</receiver>
   <slot>selectLinLogLogging(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>683</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileLog</sender>
   <signal>clicked(bool)</signal>
   <receiver>GeneralSettings</receiver>
   <slot>selectFileLogging(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectDemomode(bool)</slot>
  <slot>setControlDevice(QModelIndex)</slot>
  <slot>selectLinLogLogging(bool)</slot>
  <slot>selectFileLogging(bool)</slot>
  <slot>setRigNumber()</slot>
 </slots>
</ui>
